Refactoring/day.js

```diff
@@ -0,0 +1,19 @@
+// Get day number
+
+// Step 8
+// rename identifiers, to improve readability
+
+'use strict';
+
+const daysArray = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```

Comment:
Naming:
- Use `UPPER_SNAKE` case
- Array in name is not a good approach
```suggestion
const DAYS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```